### DISCUSSION TITLE
STM32G071GBU6 support

### DISCUSSION THREAD

**Discussion by Kamoool** [stm32duino/Arduino_Core_STM32#1675] [Libraries] [answered]
Hello,
Is it possible to add support for STM32G071GBU6 chip?
Apart from that, how does it compare to STM32L432KBU6? Unfortunately L432 is not available, so I have to make a switch and looking to same or smaller package chip with DAC.

**Comment by fpistm** [comment, marked as answer]
Hi @Kamoool 

You can add it:
https://github.com/stm32duino/wiki/wiki/Add-a-new-variant-%28board%29

Default variant is here:
https://github.com/stm32duino/Arduino_Core_STM32/tree/main/variants/STM32G0xx/G071G(6-8-B)U_G081GBU
